lib/web_ui/pubspec.yaml

```diff
@@ -19,6 +19,7 @@ dev_dependencies:
   build_web_compilers: 2.7.1
   yaml: 2.2.0
   watcher: 0.9.7+12
+  webdriver: any
```

Comment:
Let's use a concrete version of `webdriver`. Otherwise different builders and different build runs can get different versions and fail for mysterious reasons.

Reply:
done.